### ISSUE TITLE
Legacy props are not working 

### ISSUE THREAD

**Issue by tchiotludo** [8398a7/action-slack#33]
When I use legacy props (like `username`,  `channel` for example), theses options are not working.

As I see, you attach these property on `attachments` but as I know, they must be on the main root object :  

https://github.com/8398a7/action-slack/blob/533bca358f449de7c85d59d2b4df9a5040a79919/src/client.ts#L81

**Comment by 8398a7** [comment]
I want to check the behavior here as well.
Can you share what kind of yaml can be defined to reproduce the problem?

e.g.

```yaml
      - uses: 8398a7/action-slack@v2
        with:
          status: ${{ job.status }}
          username: reproduce
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }} # legacy webhook url
        if: always()
```

**Comment by 8398a7** [comment]
I was able to reproduce the issue while adding tests in #34.
I fixed the problem in #34 and released it as v2.6.0.
Thanks for the report.

**Comment by tchiotludo** [comment]
I confirm it works well ! 

Thanks
